db/models.py

```diff
@@ -1 +1,36 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_TYPE = (
+        ("ELF", "Elf"),
+        ("DWARF", "Dwarf"),
+        ("HUMAN", "Human"),
+        ("ORK", "Ork")
+    )
+    name = models.CharField(unique=True, max_length=255, choices=RACE_TYPE)
+    description = models.TextField(null=True, blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(unique=True, max_length=255)
+    bonus = models.CharField("description of the bonus", max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
```

Comment:
The `Skill` model's ForeignKey to `Race` is missing a `related_name`. Adding a `related_name` will help in accessing related models more efficiently. Consider adding `related_name='skills'` or something similar.